src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/test_webapp_commands.py

```diff
@@ -136,6 +136,17 @@ def test_linux_webapp_quick_create_cd(self, resource_group):
         # verify the web page
         self.assertTrue('Hello world' in str(r.content))
 
+    @ResourceGroupPreparer()
+    def test_create_in_different_group(self, resource_group, resource_group_location):
+        resource_group2 = resource_group + '2'
```

Comment:
This may not work when the resource_group is a string of the maximum allowed length of a resource group name. The right way to implement this test is to have two resource group preparers inject different group name to the test method.